assets/javascripts/discourse/components/explorer-schema/one-table.js

```diff
@@ -0,0 +1,17 @@
+import Component from "@glimmer/component";
+import { bind } from "discourse-common/utils/decorators";
+import { tracked } from "@glimmer/tracking";
+
+export default class ExplorerSchemaOneTable extends Component {
+  @tracked open = this.args.table.open;
+
+  get styles() {
+    return this.open ? "open" : "";
+  }
+
+  @bind
+  setTableOpen(value) {
+    event.preventDefault();
+    this.open = !value;
+  }
```

Comment:
If the only thing passed to this action is `this.open` I'd rename it to e.g. `toggleOpen` and remove the argument.